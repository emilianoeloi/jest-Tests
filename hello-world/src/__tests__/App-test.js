import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';

it("Should have App Component", () => {
  const app = shallow(
    <App />
  );
  expect(app).notNull();
});
