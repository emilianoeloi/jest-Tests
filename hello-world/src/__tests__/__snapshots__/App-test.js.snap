exports[`App.js Should have App Component 1`] = `
<div
  className="App">
  <div
    className="App-header">
    <img
      alt="logo"
      className="App-logo"
      src="logo" />
    <h2>
      Welcome to React
    </h2>
  </div>
  <p
    className="App-intro">
    To get started, edit 
    <code>
      src/App.js
    </code>
     and save to reload.
  </p>
</div>
`;
